server/profileGet.php

```diff
@@ -0,0 +1,31 @@
+<?php
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->hasPermission('can_use_admin_tools')) {
+  header("Location: /unauthorized");
+  die();
+}
+require_once 'config.php';
+getProfile($USER->getUserId());
+function getProfile($data) {
+  GLOBAL $DB_CONN:
+  $userGet = "SELECT firstName, lastName, phoneNumber, email, preparesTaxes
+  FROM User
+  WHERE userId = ? ";
+  $stmt = $DB_CONN->prepare($userGet);
+  $stmt->execute(array($data));
+  $response = $stmt->fetchAll(PDO::FETCH_ASSOC);
+  echo json_encode($response);
+
+$userabilityId = $DB_CONN->lastInsertId();
```

Comment:
I think this is some dead code leftover from your first PR. But you probably shouldn't be doing anything related to lastInsertId in this GET. 

Also, the line below this has abilityId twice in the SELECT. But like I said, I think this is leftover and will change either way once you get the abilities part working (which is secondary to shifts btw)